readme.md

```diff
@@ -38,7 +42,9 @@ Describe the differences between a SQL and NoSQL DB, and when you might use each
 
 Your Answer:
 ```text
-
+No SQL DBs are more flexible, making them appropriate for less complex
+relationships. An SQL DB is more rigid, but it also allows for easier
+many-to-many relationships.
```

Comment:
:+1: Definitely on the flexibility and the ease of many-to-many relationships but can model very complex relationships with NoSQL databases